src/Sylius/Behat/Page/Admin/Crud/IndexPage.php

```diff
@@ -76,6 +76,14 @@ public function isResourceOnPage(array $parameters)
     }
 
     /**
+     * @return int
+     */
+    public function countTableRecords()
```

Comment:
We should not assume that it is a table. Index page contains a collection (paginated or not) of items, so we should have something like ``countItems``?